be2-scala/src/main/scala/ch/epfl/pop/model/objects/Channel.scala

```diff
@@ -51,6 +51,7 @@ object Channel {
   final val rootChannel: Channel = Channel(s"${SEPARATOR}root")
   final val rootChannelPrefix: String = s"${SEPARATOR}root${SEPARATOR}"
   private final def channelRegex: String = "^/root(/[^/]+)*$"
+  final val LAODATALOCATION: String = Channel.SEPARATOR + "data"
```

Comment:
AAAAAAAAAAAH ^^'. Let's keep our styling conventions :) I'd suggest going for `LAO_DATA_LOCATION` (all caps since constant). Moreover, you could also apply the same treatmet to rootChannel and rootChannelPrefix. At the beginning, I don't believe they were supposed to be used elsewhere (other than this file), thus probably why they look like simple private variables

Reply:
Can I do a separate PR directly after this one to refactor these, as the rootChannel etc. are also used in my other PR and it would be confusing to change things in both at the same time?

Reply:
(for rootChannel and rootChannelPrefix, I'll change LAO_DATA_LOCATION here of course)